cmd/utils/flags.go

```diff
@@ -454,17 +454,22 @@ var (
 		Name:  "allow-insecure-unlock",
 		Usage: "Allow insecure account unlocking when account-related RPCs are exposed by http",
 	}
-	RPCGlobalGasCap = cli.Uint64Flag{
+	RPCGlobalGasCapFlag = cli.Uint64Flag{
 		Name:  "rpc.gascap",
 		Usage: "Sets a cap on gas that can be used in eth_call/estimateGas (0=infinite)",
 		Value: eth.DefaultConfig.RPCGasCap,
 	}
-	RPCGlobalTxFeeCap = cli.Float64Flag{
+	RPCGlobalTxFeeCapFlag = cli.Float64Flag{
 		Name:  "rpc.txfeecap",
 		Usage: "Sets a cap on transaction fee (in ether) that can be sent via the RPC APIs (0 = no cap)",
 		Value: eth.DefaultConfig.RPCTxFeeCap,
 	}
 	// Logging and debug settings
+	VerbosityFlag = cli.IntFlag{
+		Name:  "verbosity",
+		Usage: "Logging verbosity: 0=silent, 1=error, 2=warn, 3=info, 4=debug, 5=detail",
+		Value: eth.DefaultConfig.Verbosity,
+	}
```

Comment:
This PR doesn't seem to remove the old flag in `debug/flags.go`. It seems a bit odd two have two flags defined on the same cmdline switch

Reply:
The flag is the same because the key is the name attribute, 'verbosity' in this case. VerbosiryFlag is just an helper variable to keep this flag in the same cycle of read/write/marshal/unmarshal of the config file.

Reply:
RIght -- but is the lowercase one in debug/flags.go still needed? Why not use just the `VerbosityFlag` ?

Reply:
Now I got it the question. That is because VerbosityFlag is in the `utils` package but I cannot use it directly from the `debug` package because of th error
```bash
package github.com/ethereum/go-ethereum/cmd/geth
        imports github.com/ethereum/go-ethereum/cmd/utils
        imports github.com/ethereum/go-ethereum/eth
        imports github.com/ethereum/go-ethereum/node
        imports github.com/ethereum/go-ethereum/internal/debug
        imports github.com/ethereum/go-ethereum/cmd/utils: import cycle not allowed
```